nbs/00_data.ipynb

```diff
@@ -3253,6 +3266,8 @@
     "    if len(self.tfm_list) > 0:\n",
     "        self.tfm_setup(dtype=self.dtype_in, lvl=lvl)\n",
     "        self.dc_shape = self.pipeline(self.calibration[\"flat_field_pic\"]).shape\n",
+    "    elif lvl == -1:\n",
```

Comment:
I actually don't get the need for this, the else case should make dc_shape valid?  But I don't think it matter much.